RDKShell/RDKShell.cpp

```diff
@@ -4590,6 +4615,7 @@ namespace WPEFramework {
             string method = "status";
             Core::JSON::ArrayType<PluginHost::MetaData::Service> joResult;
             auto thunderController = getThunderControllerClient();
+            //FIXME: if something is checkpointed, API hangs here
```

Comment:
Investigation is ongoing why it is hanging. It seems that inner implementations of https://github.com/rdkcentral/Thunder/blob/master/Source/WPEFramework/ControllerJsonRpc.cpp#L357 tries to talk with checkpointed app, but still not clear at what point